doc/mgmt/Show_Running_Configuration_HLD.md

```diff
@@ -0,0 +1,495 @@
+
+## **SONiC Management Framework CLI 'Show Running Configuration'**
+
+
+
+#### High Level Design Document
+
+**Rev 0.1**
+
+#### Table of Contents
+
+-   [[List of Tables]](#list-of-tables)
+
+-   [[Revision]](#revision)
+
+-   [[About This Manual]](#about-this-manual)
+
+-   [[Scope]](#scope)
+
+-   [[Definition/Abbreviation]](#definitionabbreviation)
+
+#### List of Tables
+
+[[Table 1: Abbreviations]](#table-1-abbreviations)
+
+#### About this Manual
+
+This document states the functional requirements and high level design of the new feature to support "show running-configuration" command.
+
+#### Scope
```

Comment:
Make clear that the scope is limited to Klish and not other CLIs in the system (e.g. Click, vtysh). It is also limited to configuration that is found in the CONFIG_DB.